SuperBuild/CTestConfig.cmake

```diff
@@ -0,0 +1,14 @@
+## This file should be placed in the root directory of your project.
+## Then modify the CMakeLists.txt file in the root directory of your
+## project to incorporate the testing dashboard.
+##
+## # The following are required to submit to the CDash dashboard:
+##   ENABLE_TESTING()
+##   INCLUDE(CTest)
+
+set(CTEST_PROJECT_NAME "SuperElastix")
```

Comment:
Is the name deliberately the same as the name in the root folder? Or should it be e.g. "SuperElastixSuperBuild" here?

Reply:
Good question. At least, set(CTEST_PROJECT_NAME "SuperElastix") + set(CTEST_DROP_LOCATION "/submit.php?project=SuperElastix") seems a bit redundant: two times "SuperElastix". I see at https://cmake.org/Wiki/CTest:Buildserver they use ${CTEST_PROJECT_NAME} within their set(CTEST_DROP_LOCATION ...). However, that's just a style consideration. Clearly it's common for CTEST_PROJECT_NAME to match the CDash entry. And as long as it works the way we do it now (both for SuperBuild and SuperElastix-test), I think it's fine.

Reply:
Yes let's leave it this way.